src/utils/constants/URLs.js

```diff
@@ -0,0 +1,4 @@
+export const CONDITIONS_OF_USE_URL = 'https://cdss.ca.gov/conditions-of-use';
+export const FEEDBACK_EMAIL = 'mychildcare@dss.ca.gov';
+export const PRIVACY_POLICY_URL = 'https://cdss.ca.gov/privacy-policy';
+export const REGISTER_TO_VOTE_URL = 'https://registertovote.ca.gov/';
```

Comment:
No change required, just a note on how I do this. I prefer to name this kind of file to `index.js`, and then I import like this:

```
import { FEEDBACK_EMAIL } from '../utils/constants';
```

Pro: less typing to import.